tests/unit/helpers/number-format-test.js

```diff
@@ -0,0 +1,17 @@
+import { numberFormat } from 'dummy/helpers/number-format';
+import { module, test, skip } from 'qunit';
+
+module('Unit | Helper | number format');
+
+// Works in Chrome but doesn't work in terminal
```

Comment:
This is a test of a helper that just calls number.toLocaleString(), so that is all we are testing here.  It looks like phantomJS does not apply the locale formatting correctly. https://github.com/ariya/phantomjs/issues/12581. I think it works fine across browsers.  I modified the test per someone's suggestion on that issue.  Skipping the test if it's running in PhantomJS but running elsewhere.